apps/prairielearn/src/pages/instructorCourseAdminStaff/instructorCourseAdminStaff.html.ts

```diff
@@ -0,0 +1,748 @@
+                    class="btn btn-light btn-sm ml-auto"
+                    id="coursePermissionsDeleteNonOwnersButton"
+                    data-toggle="popover"
+                    data-container="body"
+                    data-html="true"
+                    data-placement="auto"
+                    title="Delete non-owners"
+                    data-content="${escapeHtml(
+                      CoursePermissionsDeleteNonOwnersForm({
+                        csrfToken: resLocals.__csrf_token,
+                        id: 'coursePermissionsDeleteNonOwnersButton',
+                      }),
+                    )}"
+                    data-trigger="manual"
+                    onclick="$(this).popover('show')"
+                  >
+                    <i class="fas fa-users-slash" aria-hidden="true"></i>
+                    <span class="d-none d-sm-inline">Delete non-owners</span>
+                  </button>
+                  <button
+                    type="button"
+                    class="btn btn-light btn-sm ml-auto"
+                    id="coursePermissionsInsertButton"
+                    data-toggle="popover"
+                    data-container="body"
+                    data-html="true"
+                    data-placement="auto"
+                    title="Add users"
+                    data-content="${escapeHtml(
+                      coursePermissionsInsertForm({
+                        csrfToken: resLocals.__csrf_token,
+                        id: 'coursePermissionsInsertButton',
+                        uidsLimit,
+                        courseInstances,
+                      }),
+                    )}"
+                    data-trigger="manual"
+                    onclick="$(this).popover('show')"
+                  >
+                    <i class="fas fa-users" aria-hidden="true"></i>
+                    <span class="d-none d-sm-inline">Add users</span>
+                  </button>
+                </div>
+              </div>
+            </div>
+            ${StaffTable({
+              csrfToken: resLocals.__csrf_token,
+              courseUsers,
+              authnUser: resLocals.authn_user,
+              user: resLocals.user,
+              isAdministrator: resLocals.is_administrator,
+            })}
+            <small class="card-footer">
+              ${hasUnknownUsers
+                ? html`
+                    <p class="alert alert-warning">
+                      Users with name "<span class="text-danger">Unknown user</span>" either have
+                      never logged in or have an incorrect UID.
+                    </p>
+                  `
+                : ''}
+              <details>
+                <summary>Recommended access levels</summary>
+                ${AccessLevelsTable()}
+              </details>
+            </small>
+          </div>
+        </main>
+      </body>
+    </html>
+  `.toString();
+}
+
+function coursePermissionsRemoveStudentDataAccessForm({
```

Comment:
Guess what my comment is going to be (`PascalCase`, both here and below 😉)

Reply:
One day I'll get these right. Updated to `PascalCase` in [38466f5](https://github.com/PrairieLearn/PrairieLearn/pull/9822/commits/38466f5b501777db47a1f16f8668ec8e2162777a).